db/migrate/20231124002142_remove_icon_from_categories.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveIconFromCategories < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :categories, :icon, :string
+  end
+end
```

Comment:
- [ ] [REQUIRED] There is another migration error here @diarisdiakite and the error indicates that there is an attempt to remove a column named "icon" from the "categories" table in your database, but the column "icon" does not exist in that table. This results in an undefined column error.

<img width="737" alt="Screenshot 2023-12-02 at 20 13 19" src="https://github.com/diarisdiakite/my-account/assets/58692590/b7c4e478-29cd-40ff-b9df-e5acdad923fb">

### To resolve this error
- Kindly remove this migration file manually and run a rollback as suggested earlier.  Then run a migration. That was how I was able to get the application working.